safe/gui/tools/wizard/step_kw57_extra_keywords.py

```diff
@@ -77,6 +80,26 @@ def set_widgets(self):
             volcano_name_checkbox = QCheckBox(tr('Volcano Name'))
             volcano_name_line_edit = QLineEdit()
 
+            # Volcano longitude
+            volcano_longitude_checkbox = QCheckBox(
+                tr('Volcano longitude'))
+            volcano_longitude_spin_box = QDoubleSpinBox()
```

Comment:
I would personally go for all QgsX widgets, like QgsDoubleSpinBox QgsDateTimeEdit ... small comment ;-)

Reply:
Ah yes, I should do that, but I usually forget that we have QgsX widget, and just simply go with QX widget. I will try to do it next time.

Reply:
No worries. Me too.
I thinking about that because they improved all these QgsWidgets quite a lot in QGIS 3 and we will have the same UI as QGIS Desktop. For QgsSpinbox, they improved if a user scroll on them (if you want to move up or down the widget or scroll the value in the spin widget